docs/en/latest/plugins/limit-conn.md

```diff
@@ -23,11 +23,12 @@ title: limit-conn
 
 ## Summary
 
-- [**Name**](#name)
-- [**Attributes**](#attributes)
-- [**How To Enable**](#how-to-enable)
-- [**Test Plugin**](#test-plugin)
-- [**Disable Plugin**](#disable-plugin)
+- [Summary](#summary)
```

Comment:
The summary usually doesn't count itself.

Reply:
my bad, automatically modified by the editor